plugins/modules/group.py

```diff
@@ -86,6 +86,11 @@
         - Set this to an empty list to remove all members from a group.
         type: list
         elements: raw
+  flatten:
+    description:
+    - Flattens nested groups.
+    type: bool
+    default: false
```

Comment:
Needs a `version_added: 1.7.0` to indicate when this option was added.

Reply:
Done